src/frontend/.gitignore

```diff
@@ -22,3 +22,4 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+*.scssc
```

Comment:
based on @binggaofw's feedback, lets collapse the gitignore in the frontend to a .gitignore at the project root directory.

thoughts?v

Reply:
I'm on it.